packages/react/cypress/plugins/index.js

```diff
@@ -80,6 +80,10 @@ const webpackConfig = {
           },
         ],
       },
+      {
+        test: /\.mdx?$/,
```

Comment:
What is this for?

Reply:
cypress needs to know how to process mdx files, with the stateful table test, because mdx is imported in that file.